### DISCUSSION TITLE
What's the maximum number of stored Query History / Saved Queries? Can this number be increased? Thanks!

### DISCUSSION THREAD

**Discussion by bonzi-123** [tprouvot/Salesforce-Inspector-reloaded#526] [Q&A] [answered]
I wanted to know what the maximum number of stored Query History / Saved Queries is, and if this number can be increased. I have queries from 6 months ago that I sometimes have to re-use and I no longer see them in the Query History. Thanks

**Comment by tprouvot** [comment, marked as answer]
Hi @bonzi-123 ,
The limit is 100 queries, I can add a new option to let users setup the limit

**Reply by bonzi-123** [reply]
Hi! Yes, that would be great! Can you allow the user to set the limit both for Query History & Saved Queries? Thanks!

**Comment by tprouvot** [comment]
Dev is done, this will be part of the 1.25 and before on the beta version 🚀
